### ISSUE TITLE
grey_accessibilityValue not working

### ISSUE THREAD

**Issue by ilahomesick** [google/EarlGrey#1448]
Hi, I'm trying to use the accessibilityValue of a cell instead of Accessibility ID because i need to use it for a different purpose, I've set the code as accessibility value and i can also see it using the accessibility inspector, but when i call the grey_accessibilityValue function, it doesn't work and the test fails.
I'm 100% sure that the cell is there and the code I'm passing is correct, because if i use grey_accessibilityID instead and I assign the code to accessibilityIdentifier, it works without any problem.

<img width="818" alt="Screenshot 2020-09-17 at 11 37 14" src="https://user-images.githubusercontent.com/15572597/93456507-0fd0ed00-f8de-11ea-802a-62b35d57ea53.png">
<img width="568" alt="Screenshot 2020-09-17 at 11 37 25" src="https://user-images.githubusercontent.com/15572597/93456521-13647400-f8de-11ea-871b-adce60f951ea.png">
<img width="580" alt="Screenshot 2020-09-17 at 11 37 43" src="https://user-images.githubusercontent.com/15572597/93456522-13fd0a80-f8de-11ea-8aed-1b62382c03c1.png">
